pasqal_cloud/batch.py

```diff
@@ -107,20 +114,25 @@ def jobs(self) -> Dict[str, Job]:
         )
         return {job.id: job for job in self.ordered_jobs}
 
-    @validator("configuration", pre=True)
+    @jobs.setter
+    def jobs(self, _: Any) -> None:
+        # Does not set jobs attribute as it is built from 'ordered_jobs'
```

Comment:
I'm not totally convinced by my explanation, if you have a better manner of explaining it, I am all ears.